services/web/server/src/simcore_service_webserver/session/access_policies.py

```diff
@@ -72,7 +74,7 @@ async def _wrapper(request: web.Request):
 
             response = await handler(request)
 
-            if response.status < 400:  # success
+            if response.status < _HTTP_MIN_ERROR_CODE:
```

Comment:
I actually kind of agree, that you could just do ```status < status.CLIENT_ERROR.status_code``` would be the same and clear.

Reply:
Since two people align, I will not fight it but just to be clear:  here the important thing is NOT to know which error is but rather to stress that this is the code that limits between error and no-error!

Reply:
I think what you say is clear. what is unclear is why you use 400 instead of the status name. it is pretty clear that 400 is the first possible error.